### PR TITLE
Rm comp dep

### PR DESCRIPTION
**Pull request by bjzhjing**
## Description

Remove fixed dependencies for common microservices to allow end user to flexibly configure their workload pipeline.

## Issues

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

Enhancement for existing helm-chart design.

## Dependencies

n/a

## Tests

Issue 'helm install chatqna chatqna' to install chatqna workload to verify.

### CODE REVIEW

**Review by lianhao — CHANGES_REQUESTED** [review]
Please also adjust the helm value files in all corresponding charts, otherwise parent helm chart's values won't be able to passed into child chart correctly.

**Comment by yongfengdu** [comment]
closes: #147

**Review by lianhao — APPROVED** [review]

File: helm-charts/chatqna/gaudi-values.yaml [hunk 1]
```diff
@@ -31,16 +31,17 @@ securityContext:
 
 # To override values in subchart llm-uservice
 llm-uservice:
+  TGI_LLM_ENDPOINT: ""
```
Comment (lianhao) [inline helm-charts/chatqna/gaudi-values.yaml:34]: maybe we should not set any thing for llm-uservice here, if the default one works for us

File: helm-charts/chatqna/values.yaml [hunk 2]
```diff
@@ -31,10 +31,11 @@ securityContext:
 
 # To override values in subchart llm-uservice
 llm-uservice:
+  TGI_LLM_ENDPOINT: ""
```
Comment (lianhao) [inline helm-charts/chatqna/values.yaml:34]: ditto

**Review by lianhao — APPROVED** [review]

**Review by daisy-ycguo — APPROVED** [review]
lgtm

**Pull request merged**